app/controllers/AuthorizedToViewController.scala

```diff
@@ -104,11 +104,15 @@ class AuthorizedToViewController @Inject()(authenticate: IdentifierAction,
     val searchQuery = stripWithWhitespace(query)
 
     val result = for {
-      cdsAccounts: CDSAccounts <- apiService.getAccounts(request.user.eori)
+      gbEoriAccounts: CDSAccounts <- apiService.getAccounts(request.user.eori)
       xiEORI: Option[EORI] <- dataStoreService.getXiEori(request.user.eori)
+
```

Comment:
Could you please remove this line break to improve readability inside for yield?
Same for line no 111

Reply:
Code chane done